plugins/module_utils/zos_mvs_raw.py

```diff
@@ -99,3 +129,308 @@ def __init__(self, rc, stdout, stderr):
+            file_data_type : str, optional
+                           The type of data that is (or will be) stored in the UNIX file. Defaults to None.
+            record_length : int, optional
+                          The specified logical record length for the UNIX file. Defaults to None.
+            block_size : int, optional
+                       The specified block size for the UNIX file being allocated. Defaults to None.
+            record_format : str, optional
+                          The specified record format for the UNIX file. Defaults to None.
+            return_content : dict, optional
+                           Determines how content should be returned to the user. Defaults to None.
+        """
+        self.return_content = ReturnContent(**(return_content or {}))
+        super().__init__(
+            path_name=path,
+            normal_disposition=disposition_normal,
+            conditional_disposition=disposition_abnormal,
+            path_mode=mode,
+            access_group=access_group,
+            status_group=status_group,
+            file_data=file_data_type,
+            record_length=record_length,
+            block_size=block_size,
+            record_format=record_format,
+        )
+
+
+class RawInputDefinition(InputDefinition):
+    """Wrapper around InputDefinition to contain information about
+    desired return contents.
+    """
+
+    def __init__(
+            self,
+            content="",
+            return_content=None,
+            tmphlq="",
+            **kwargs
+    ):
+        """
+        InputDefinition (InputDefinition): Input DD data type to be used in a DDStatement.
+        Parameters
+        ----------
+            content : str
+                    The content to write to temporary data set / stdin.
+            return_content : dict, optional
+                           Determines how content should be returned to the user. Defaults to {}.
+        """
+        self.return_content = ReturnContent(**(return_content or {}))
+        super().__init__(
+            content=content,
+            tmphlq=tmphlq)
+
+
+class RawOutputDefinition(OutputDefinition):
+    """Wrapper around OutputDefinition to contain information about
+    desired return contents.
+    """
+
+    def __init__(
+            self,
+            return_content=None,
+            tmphlq="",
+            **kwargs
+    ):
+        """
+        OutputDefinition (OutputDefinition): Output DD data type to be used in a DDStatement.
+        Parameters
+        ----------
+            content : str
+                    The content to write to temporary data set / stdin.
+            return_content : dict, optional
+                           Determines how content should be returned to the user. Defaults to {}.
+        """
+        self.return_content = ReturnContent(**(return_content or {}))
+        super().__init__(
+            tmphlq=tmphlq
+        )
+
+
+class ReturnContent(object):
```

Comment:
Could you share samples of how this ReturnContent changes the way dd contents are returned?